README.md

```diff
@@ -157,6 +157,12 @@ To skip cleaning the project:
 snapshot --noclean
 ```
 
+To only run tests (i.e. don't actually take any screenshots):
```

Comment:
This will still take screenshots, but doesn't transfer them to the local directory, right?